### ISSUE TITLE
Query string being passed to every request in the chain

### ISSUE THREAD

**Issue by erdalkiran** [basti1302/traverson#32]
Hi,
I like to know how I pass query string only to the link I follow not the base link.
For example: 

```javascript
var baseUrl = 'http://api.test.com/resource';
var api = traverson.jsonHal.from(baseUrl);

api.newRequest()
    .follow('follow_link')
    .withRequestOptions({
        headers: {
            'Content-Type': 'application/hal+json',
            'Accept': 'application/hal+json'
        },
        qs: {
            'sortBy': 'date'
        }
    })
    .getResource(function (error, document) {
        //do something
    });
```
If i run the sample code above, query string parameter is also passed to the first request.
Like http://api.test.com/resource?sortBy=date
How can I prevent this situation? I only want query string to be passed to follow_link

Thnx,
Erdal

**Comment by basti1302** [comment]
Currently, this is not possible. I'll leave this open as a feature request.

**Comment by basti1302** [comment]
Actually, this seems to be a duplicate of #25, so I'm closing it.

**Comment by basti1302** [comment]
#25 has landed in 1.1.0
